internal/nodes/avi_model_evh_nodes.go

```diff
@@ -574,6 +578,9 @@ func (o *AviObjectGraph) ConstructAviL7SharedVsNodeForEvh(vsName, key string, ro
 		VrfContext: vrfcontext,
 	}
 
+	if lib.GetT1LRPath() != "" {
+		vsVipNode.T1Lr = lib.GetT1LRPath()
```

Comment:
minor: this can go in with the above conditional itself.

Reply:
I think if I move this, then I have to shuffle a bit more of the code. the vsVipNode is only declared after that previous conditional.

Reply:
ah yes, i did not see the vs/vsvip difference, my bad.